.github/workflows/reusable_cve_scan.yml

```
@@ -0,0 +1,211 @@
+          image_tag=${output#Loaded image: }
+          echo "IMAGE_TAG=$image_tag" >> $GITHUB_ENV
+
+      - name: Create Python Environment
+        run: |
+          python3 -m venv env 
+          source env/bin/activate
+          pip install requests tqdm
+          
+      - name: Install Trivy
+        if: ${{ inputs.run_trivy }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:102: Quote this to prevent word splitting

(shellcheck)